### DISCUSSION TITLE
hainanu-course-comments/6.%E4%B8%93%E4%B8%9A%E9%80%89%E4%BF%AE%E8%AF%BE/MATLAB%E4%B8%8E%E6%99%BA%E8%83%BD%E7%B3%BB%E7%BB%9F%E4%BB%BF%E7%9C%9F/

### DISCUSSION THREAD

**Discussion by giscus** [beiyuouo/hainanu-course-comments#5] [General]
# hainanu-course-comments/6.%E4%B8%93%E4%B8%9A%E9%80%89%E4%BF%AE%E8%AF%BE/MATLAB%E4%B8%8E%E6%99%BA%E8%83%BD%E7%B3%BB%E7%BB%9F%E4%BB%BF%E7%9C%9F/

https://www.bj-yan.top/hainanu-course-comments/6.%E4%B8%93%E4%B8%9A%E9%80%89%E4%BF%AE%E8%AF%BE/MATLAB%E4%B8%8E%E6%99%BA%E8%83%BD%E7%B3%BB%E7%BB%9F%E4%BB%BF%E7%9C%9F/

**Comment by Iriszyh** [comment]
考核形式开卷考试。
老师讲的不错，上课要带电脑，就老师讲一些代码，然后我们在电脑上复现一下。实验课就是课本课后题。老师挺不错
